content/events/summit-brazil-24/index.md

```diff
@@ -0,0 +1,54 @@
+---
+title: "[Upcoming Event] Espressif Summit Brazil 2024"
+date: 2024-06-10
+draft: false
+description: "Espressif Systems Summit in Brazil"
+tags: ["event", "announcement"]
+showDate : false
+---
+
+## Espressif Summit Brazil 2024
+
+We are pleased to announce the first in-person Espressif event in Brazil, taking place in Campinas from July 17th to 18th, from 9 am to 6 pm. On the first day, we will present talks about our most recent products, including the ESP32-P4, ESP32-C5, ESP-IDF, Nuttx, Zephyr, low-power applications, Arduino, MQTT, and more.
+
+On the second day, workshops will be presented in two different tracks, including "Introduction to ESP-IDF with the ESP32-C6," "IoT Workshop with RainMaker and Matter," and "Zephyr with ESP32." All the talks and workshops will be presented by Espressif experts and will be mostly in **Portuguese**.
```

Comment:
The paragraphs combine unrelated things. Suggest splitting information among the paragraphs like this:

We are pleased to announce the first in-person Espressif event in Brazil, taking place in Campinas from July 17th to 18th, from 9 am to 6 pm.

The activities are as follows:
  - Day 1: we will present talks about our most recent products, including the ESP32-P4, ESP32-C5, ESP-IDF, Nuttx, Zephyr, low-power applications, Arduino, MQTT, and more.
  - Day 2: workshops will be presented in two different tracks, including "Introduction to ESP-IDF with the ESP32-C6," "IoT Workshop with RainMaker and Matter," and "Zephyr with ESP32."

All the talks and workshops will be presented by Espressif experts and will be mostly in **Portuguese**.